app/src/main/java/com/example/composetemplate/presentation/common/FullScreenProgressBar.kt

```diff
@@ -21,7 +22,7 @@ fun FullScreenProgressBar(isLoading: Boolean) {
             contentAlignment = Alignment.Center
         ) {
             CircularProgressIndicator(
-                color = Color.White, // Set color as needed
+                color = CustomTheme.colors.text,
```

Comment:
why color named text on indicator?

Reply:
The application uses a universally agreeable color for text, making it easier for us to set colors for both dark and light modes.